packages/payment-detection/src/near-info-retriever.ts

```diff
@@ -55,7 +55,6 @@ export class NearInfoRetriever {
         COALESCE(a.args::json->>'deposit', '') as deposit,
         COALESCE(a.args::json->>'method_name', '') as method_name,
         COALESCE((a.args::json->'args_json')::json->>'to', '') as "to",
```

Comment:
Pending on @BeskydAbc review

Reply:
Instead of 'Amount' field, you need to use the 'Deposit' field
[https://github.com/near-request/request-network/commit/c03933c6eea0c7bcb66d325d311f26feb3052c6b#diff-f54470a15c95ccf65e878836b9980fd6c72eca557bad8fe160022bdd0d8c4219]